index.html

```diff
@@ -55,5 +59,236 @@ <h4 class="headline-sm-suport">LET'S CONNECT</h4>
     </section>
     <!--headline section ends-->
 
+    <!--Card works starts-->
+    <section class="works">
```

Comment:
- [ ] Kindly consider using display: grid in this section as mentioned in the requiremets:
  - In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.